utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -40,6 +47,20 @@ struct ThreadStats {
     classes: HashSet<&'static str>,
 }
 
+extern "C" {
+    pub fn gettid() -> u32;
+}
```

Comment:
I would like to suggest using `gettid` implementation from `nix` crate (https://docs.rs/nix/0.19.1/nix/unistd/fn.gettid.html) instead of depending implicitly on libc, and explicit use of unsafe calling this method

Reply:
Fixed